### PR TITLE
Manually enabled gas statistics for erc20 token test.

### PR DESCRIPTION
**Pull request by hrkrshnn**

### CODE REVIEW

**Review by axic — DISMISSED** [review]

File: test/libsolidity/semanticTests/various/erc20.sol [hunk 1]
```diff
@@ -94,14 +94,37 @@ contract ERC20 {
...
+// gas legacyOptimized: 23368
 // totalSupply() -> 20
+// gas irOptimized: 23446
+// gas legacy: 23524
+// gas legacyOptimized: 23368
 // transfer(address,uint256): 2, 5 -> true
 // ~ emit Transfer(address,address,uint256): #0x1212121212121212121212121212120000000012, #0x02, 0x05
+// gas irOptimized: 48731
+// gas legacy: 49317
+// gas legacyOptimized: 48491
```
Comment (axic) [inline test/libsolidity/semanticTests/various/erc20.sol:112]: Pretty cool that most of the costs are very close, ~1% increase.

File: test/libsolidity/semanticTests/various/erc20.sol [hunk 2]
```diff
@@ -94,14 +94,37 @@ contract ERC20 {
     }
 }
 // ====
-// compileViaYul: true
+// compileViaYul: also
 // ----
+// constructor()
+// ~ emit Transfer(address,address,uint256): #0x00, #0x1212121212121212121212121212120000000012, 0x14
+// gas irOptimized: 23446
+// gas legacy: 833310
```
Comment (axic) [inline test/libsolidity/semanticTests/various/erc20.sol:102]: Curious: what is being generated here?! 👀

Comment (hrkrshnn) [reply test/libsolidity/semanticTests/various/erc20.sol:102]: Actually, the other numbers here seem to be wrong :( I'l check why.

Comment (hrkrshnn) [reply test/libsolidity/semanticTests/various/erc20.sol:102]: The numbers now should be correct. But isoltest has some trouble auto updating it, for constructor. I'll just make a issue for now.

**Review by axic — APPROVED** [review]

**Pull request merged**